tool/release/createnotes/Note.kt

```diff
@@ -120,7 +120,7 @@ ${others.map(Note::toMarkdown).joinToString("\n")}
 }
 
 fun collectNotes(org: String, repo: String, commits: List<String>, githubToken: String): List<Note> {
-    return commits.flatMap { commit ->
+     return commits.reversed().flatMap { commit ->
```

Comment:
I did this here, but pushed a commit to master to do this in a better place as a follow up. The commits appear to be in time-order, so reversing them puts most-recent first, which is how we want the release-notes made

Reply:
Hmm why would reverse order be better?

Reply:
because we always list our release notes with most recent first, not most recent last (well, at least this is how we used to do it!) I found it more intuitive for some reason